### PR TITLE
[530] Coupon Banner

### PR DESCRIPTION
**Pull request by rlaco-qualityunit**
**Changes proposed in this Pull Request**
New assets for coupons discount banner
Template is saved on production web as a "Countdown Banner"

**Testing instructions**
Test banner on desktop and mobile

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#530

### CODE REVIEW

File: assets/styles/components/_BlockCoupon.scss [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+		span::before {
+			display: none;
+		}
+
+		svg {
+			width: 1em;
+			margin-right: 0.5em;
+
+			path {
```
Comment (jremen) [inline assets/styles/components/_BlockCoupon.scss:199]: use the fill on whole SVG

File: assets/styles/components/_BlockCoupon.scss [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+			&-label {
+				font-size: 0.4375em;
+				text-align: center;
+			}
+		}
+
+		&-background-overlay {
+			display: none;
+			background-position: center center;
```
Comment (jremen) [inline assets/styles/components/_BlockCoupon.scss:155]: Why is this here when no color or image is defined?

**Review by jremen — CHANGES_REQUESTED** [review]
Make fixes according to the comments

File: assets/styles/components/_BlockCoupon.scss (hunk 2 above)
Comment (rlaco-qualityunit) [reply assets/styles/components/_BlockCoupon.scss:155]: Image is defined by user as overlay in Elementor block and is not necessary to display it on small displays.
(gradient is defined in section CSS by default and can be overwritten by user in Elementor)

File: assets/styles/components/_BlockCoupon.scss (hunk 1 above)
Comment (rlaco-qualityunit) [reply assets/styles/components/_BlockCoupon.scss:199]: fixed

**Comment by rlaco-qualityunit** [comment]
> Make fixes according to the comments

@jremen done

**Review by mvstepanek — APPROVED** [review]

**Pull request merged**